packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Carbon from '../../../../carbon-graphs/src/js/carbon';
+import '../../css/Graph.module.scss';
+import '../../css/LineGraph.module.scss';
+import utils from '../../../../carbon-graphs/src/js/helpers/utils';
+
+class LineGraph extends React.Component {
```

Comment:
At this point I'd recommend writing functional components with hooks: https://reactjs.org/docs/hooks-effect.html

```jsx
const LineGraph = ({ graphID, graphConfig, dataset }) => {
  React.useEffect(() => {
      const graph = Carbon.api.graph(this.props.graphConfig);
      (utils.isArray(this.props.dataset)) ? (this.props.dataset.forEach((data) => {
        graph.loadContent(Carbon.api.line(data));
      })) : graph.loadContent(Carbon.api.line(this.props.dataset));
  }, [graphConfig, dataset]);

  return (
    <div id="canvasContainer">
      <div id="tooltip" className="tooltip" style={{ display: 'none' }} />
      <div id={this.props.graphID} />
    </div>
  );
};


```

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/74/commits/631c1818a81503e72ba2f63e2f41eb9df667ba04#diff-5e17603406b642035820d408335769ae48391021a7787fd86404b130590e9606R14-R34).